src/data.js

```diff
@@ -59,6 +59,26 @@ module.exports = [
       'Python',
     ],
   },
+  {
+    name: 'Maxim Villivald',
+    description: 'Web Developer, IT Student 👨‍💻, Blogger & part time Tram Driver 🚃.',
+    url: 'https://proj.ninja/uses',
```

Comment:
The page don't exist

Not Found
The requested URL /uses was not found on this server.

Reply:
I am sorry, there was a strange issue with .htaccess on the server-side. Now it works.

Reply:
great, thanks.

i like the individual style of the page